Template/Tag/AddThisTag.web.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+            var siteId = parameters.get('AddThisPubId');
+            if (siteId) {
+                var script = document.createElement('script');
+                var s = document.getElementsByTagName('script')[0];
+                script.type = 'text/javascript';
+                script.src = '//s7.addthis.com/js/300/addthis_widget.js#pubid=' + siteId;
```

Comment:
Probably won't make a big difference but could potentially load it async and deferred (I reckon current implementation is what addthis suggests)

Reply:
I'm not sure if it changes anything as it only gets executed after pageload. But I guess we can add it to all tags (maybe except for error tracking).